CMakeLists.txt

```diff
@@ -330,8 +330,26 @@ resolve_dependency(Boost 1.66.0 COMPONENTS ${BOOST_INCLUDE_LIBRARIES})
 # Range-v3 will be enable when the codegen code actually lands keeping it here
 # for reference. find_package(range-v3)
 
-find_package(gflags COMPONENTS shared)
-find_package(glog REQUIRED)
+set_source(gflags)
+resolve_dependency(gflags COMPONENTS shared)
+if(NOT TARGET gflags::gflags)
+  get_target_property(gflags_global gflags IMPORTED_GLOBAL)
+  if(NOT gflags_global)
+    # Some configurations don't import the gflags target globally which will
+    # block the creation of an alias
+    set_target_properties(gflags PROPERTIES IMPORTED_GLOBAL TRUE)
+  endif()
+  add_library(gflags::gflags ALIAS gflags)
+endif()
+
+# set_source(Unwind) resolve_dependency(Unwind)`
+
+if(${gflags_SOURCE} STREQUAL "BUNDLED")
+  set(glog_SOURCE BUNDLED)
```

Comment:
Should this be gflags_SOURCE ?

Reply:
No this is used to force glog from source if we build gflags from source to avoid conflicts if the system glog was built against another gflags (which is likely). I will add a comment.